workflow/exp/exp.jet_conus3km

```diff
@@ -3,7 +3,8 @@
 #
 export NET=rrfs                    # rrfs, rtma, 
 export MESH_NAME=conus3km         # conus12km, conus3km, atl3km, etc
-export EXP_NAME=rrfsdet
+export WGF="det"   # working group function = "det", "enkf", "ensf", "firewx"
```

Comment:
@hu5970 Thanks for adding the comment!

@BenjaminBlake-NOAA @MatthewPyle-NOAA 
I don't know whether we have a chance to change `ensf` as it looks very similar to `enkf` or `ensrf` (variants of ensemble Kalman filters)
Changing to something like `enfs` may reduce some confusion. Just a random thought.

Reply:
@guoqing-noaa  Let's see if @lgannoaa has any comments on changing ensf because he is the one who added WGF to the workflow for RRFSv1.  If we were to change it, my vote would be for enfcst, which is slightly longer but makes it clear you are referring to the ensemble forecast component.

Reply:
I will recommend not to change the ensf for now, so we have clear reference. @guoqing-noaa

Reply:
@BenjaminBlake-NOAA @lgannoaa 
Thanks for the input!

If we can use `enfcst`, that will be sweeter! I can imagine `ensf` may cause lots of confusion when more people use the mpas+jedi-based rrfs-workflow. I would suggest we make this change for the `rrfs-mpas-jedi` branch as early as possible.